tests/v2/validation/provisioning/rke2/agent_customization_test.go

```diff
@@ -0,0 +1,232 @@
+//go:build validation
+
+package rke2
+
+import (
+	"testing"
+
+	"github.com/rancher/rancher/tests/v2/validation/provisioning/permutations"
+	"github.com/rancher/shepherd/clients/rancher"
+	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
```

Comment:
I spotted quite a few V3 objects throughout this review, are there any V1 equivalents that you could use to replace these objects?

Reply:
spoke offline to discuss further - The following ticket will move anything v3 to v1 - at this point, a refactor would be required for everything depending on the management object in the provisioning and clusterConfigs: https://github.com/rancher/qa-tasks/issues/915